posthog/temporal/data_imports/pipelines/salesforce/__init__.py

```diff
@@ -0,0 +1,225 @@
+        },
+        "Contact": {
+            "name": "Contact",
+            "table_name": "contact",
+            "primary_key": "id",
+            "write_disposition": "replace",
+            "endpoint": {
+                "data_selector": "records",
+                "path": "/services/data/v61.0/query",
+                "params": {
+                    "q": "SELECT FIELDS(STANDARD) FROM Contact",
+                },
+            },
+        },
+        "Campaign": {
+            "name": "Campaign",
+            "table_name": "campaign",
+            "primary_key": "id",
+            "write_disposition": "replace",
+            "endpoint": {
+                "data_selector": "records",
+                "path": "/services/data/v61.0/query",
+                "params": {
+                    "q": "SELECT FIELDS(STANDARD) FROM Campaign",
+                },
+            },
+        },
+        "Product2": {
+            "name": "Product2",
+            "table_name": "product2",
+            "primary_key": "id",
+            "write_disposition": "replace",
+            "endpoint": {
+                "data_selector": "records",
+                "path": "/services/data/v61.0/query",
+                "params": {
+                    "q": "SELECT FIELDS(STANDARD) FROM Product2",
+                },
+            },
+        },
+        "Pricebook2": {
+            "name": "Pricebook2",
+            "table_name": "pricebook2",
+            "primary_key": "id",
+            "write_disposition": "replace",
+            "endpoint": {
+                "data_selector": "records",
+                "path": "/services/data/v61.0/query",
+                "params": {
+                    "q": "SELECT FIELDS(STANDARD) FROM Pricebook2",
+                },
+            },
+        },
+        "PricebookEntry": {
+            "name": "PricebookEntry",
+            "table_name": "pricebook_entry",
+            "primary_key": "id",
+            "write_disposition": "replace",
+            "endpoint": {
+                "data_selector": "records",
+                "path": "/services/data/v61.0/query",
+                "params": {
+                    "q": "SELECT FIELDS(STANDARD) FROM PricebookEntry",
+                },
+            },
+        },
+        "Account": {
+            "name": "Account",
+            "table_name": "account",
+            "primary_key": "id",
+            "write_disposition": "merge" if is_incremental else "replace",
+            "endpoint": {
+                "data_selector": "records",
+                "path": "/services/data/v61.0/query",
+                "paginator": SalesforceIncrementalPaginator(schema="Account", subdomain=subdomain),
+                "params": {
+                    "q": {
+                        "type": "incremental",
+                        "cursor_path": "SystemModstamp",
+                        "initial_value": "SELECT FIELDS(STANDARD) FROM Account WHERE SystemModstamp > 2000-01-01T00:00:00.000+0000",
```

Comment:
Unclear how to do incremental with this rest query format. I tried to do this but initial_value needs to match the cursor_path type i believe and cursor_path in this case is a datetime.

Oddly enough, the data does return but dlt doesn't write it

Reply:
Hey @EDsCODE, 
> initial_value needs to match the cursor_path type i believe and cursor_path in this case is a datetime.

This is true. We're going to allow data conversion for the incremental params, here's the issue: https://github.com/dlt-hub/verified-sources/issues/507

Please take a look to see if the proposed solution fits. Do we miss anything?

Reply:
Yep, having a custom function to map the types/parameters would be great.

I think the suggestion in the issue would be enough for my case as we would be receiving a timestamp field and I'd be transforming it into the full query to pass back in

Reply:
@EDsCODE the PR with this feature has been merged: https://github.com/dlt-hub/verified-sources/pull/515
Please give it a go. We'll document it soon as well.